migrations/20220621153946-parkAndSubAreaRoles.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+const AWS = require('aws-sdk');
+const { runQuery, TABLE_NAME, dynamodb } = require('../lambda/dynamoUtil');
+
+const readXlsxFile = require('read-excel-file/node');
+
+const schema = {
+  'ORCS Number': {
+    prop: 'ORCS Number',
+    type: String
+  },
+  'Park': {
+    prop: 'Park',
+    type: String
+  },
+  'Park Sub Area': {
+    prop: 'Park Sub Area',
+    type: String
+  },
+  'Frontcountry Camping': {
+    prop: 'Frontcountry Camping',
+    type: String
+  },
+  'Backcountry Camping': {
+    prop: 'Backcountry Camping',
+    type: String
+  },
+  'Group Camping': {
+    prop: 'Group Camping',
+    type: String
+  },
+  'Day Use': {
+    prop: 'Day Use',
+    type: String
+  },
+  'Boating': {
+    prop: 'Boating',
+    type: String
+  },
+  'Frontcountry Cabins': {
+    prop: 'Frontcountry Cabins',
+    type: String
+  },
+  'Backcountry Cabins': {
+    prop: 'Backcountry Cabins',
+    type: String
+  },
+  'Section': {
+    prop: 'Section',
+    type: String
+  },
+  'Management Area': {
+    prop: 'Management Area',
+    type: String
+  },
+  'Bundle': {
+    prop: 'Bundle',
+    type: String
+  },
+  'Region': {
+    prop: 'Region',
+    type: String
+  },
+  'Sub Area ID': {
+    prop: 'Sub Area ID',
+    type: String
+  }
+}
+
+let parkErrors = []
+let subAreaErrors = []
+let completedParks = [];
+let completedSubAreas = [];
+
+exports.up = async function (dbOptions) {
+  await updateAllParks();
+};
+
+async function updateAllParks() {
+  try {
+    const parks = await getParks();
+
+    console.log("parks:", parks);
+
+    for(const park of parks) {
+      await processPark(park);
+    }
+  } catch (err) {
+    logger.error(err);
```

Comment:
`logger` is not declared in this migration script.

Reply:
Fixed in 0163feb